test/toM3u8.test.js

```diff
@@ -1,6 +1,7 @@
 import { toM3u8 } from '../src/toM3u8';
 import QUnit from 'qunit';
 
+QUnit.dump.maxDepth = 10;
```

Comment:
is this needed for the tests to pass? or just for the diff output when an assertion fails?

Reply:
I accidentally left this line from my debugging.